src/Eval.cpp

```diff
@@ -206,9 +207,9 @@ std::vector <std::string> Eval::getOperators ()
 std::vector <std::string> Eval::getBinaryOperators ()
 {
   std::vector <std::string> all;
-  for (unsigned int i = 0; i < NUM_OPERATORS; ++i)
-    if (operators[i].type == 'b')
-      all.push_back (operators[i].op);
+  for (const auto &opr : operators)
```

Comment:
Should reflect the implementation above, that is, (a) use `NUM_OPERATORS` constant to resize upfront and (b) more readable loop variable.

Reply:
reserving NUM_OPERATORS makes no sense as it's only doing a push_back on a subset of them.